src/xdocs/beginning_development.xml

```diff
@@ -53,7 +53,7 @@
       </p>
       <div class="wrap-content">
         <source>
-          git remote add main https://github.com/checkstyle/checkstyle
+          git remote add upstream https://github.com/checkstyle/checkstyle
```

Comment:
please make update at line `Fork your copy from Checkstyle main project. As it is described`
to `Fork Checkstyle upstream project. As it is described`

Reply:
done @romani